pyais/messages.py

```diff
@@ -1299,8 +1302,8 @@ class MessageType27(Payload):
     accuracy = bit_field(1, int, default=0)
     raim = bit_field(1, bool, default=0)
     status = bit_field(4, int, default=0, from_converter=NavigationStatus, to_converter=NavigationStatus)
-    lon = bit_field(18, int, from_converter=from_lat_lon_600, to_converter=to_lat_lon_600, default=0)
-    lat = bit_field(17, int, from_converter=from_lat_lon_600, to_converter=to_lat_lon_600, default=0)
+    lon = bit_field(18, float, from_converter=from_lat_lon_600, to_converter=to_lat_lon_600, default=0)
+    lat = bit_field(17, float, from_converter=from_lat_lon_600, to_converter=to_lat_lon_600, default=0)
     speed = bit_field(6, int, default=0)
```

Comment:
speed / course here also missing float. Add a couple more test cases for the different types to confirm. I think:
Message type 1, 9, 18, 19, 27 have speed/course. I didn't check all lat/lon usage

Reply:
No, both `speed` and `course` are unsigned integers in this case. [Source](https://gpsd.gitlab.io/gpsd/AIVDM.html#_type_27_long_range_ais_broadcast_message).

There is a matching testcase:   [def test_msg_type_27(self):](https://github.com/M0r13n/pyais/blob/e833f46cbaa157aa022113c10a5630c002104459/tests/test_decode.py#L546)

Excerpt:

```
assert msg['lon'] == 137.023333
assert msg['lat'] == 4.84
assert msg['speed'] == 57
assert msg['course'] == 167
```

which is the same as [tbsalling](http://ais.tbsalling.dk/) returns.